README.MD

```diff
@@ -1,8 +1,15 @@
 # G Bridge
```

Comment:
This is a good start, I would like to take screen shots of the steps required to set up an actions project on google as well

Reply:
This could be done in a later PR, were i would completly redo the README and add a project logo. After most of the other stuff is finished